src/CMakeLists.txt

```diff
@@ -389,6 +393,14 @@ if(BUILD_DEV)
     target_compile_definitions(migraphx PUBLIC -DBUILD_DEV)
 endif()
 
+if(MIGRAPHX_ENABLE_TENSORFLOW)
+    target_compile_definitions(migraphx PUBLIC MIGRAPHX_ENABLE_TENSORFLOW)
+endif()
+
+if(MIGRAPHX_ENABLE_ONNX)
+    target_compile_definitions(migraphx PUBLIC MIGRAPHX_ENABLE_ONNX)
```

Comment:
I would prefer a define when we are disabling instead. Something like `-DMIGRAPHX_ONNX_DISABLED`.